code/mob/living/critter/ai/shared.dm

```diff
@@ -101,6 +101,21 @@
 	var/max_path_dist = 50 //keeping this low by default, but you can override it - see /datum/aiTask/sequence/goalbased/rally for details
 	var/list/found_path = null
 	var/turf/move_target = null
+	var/owner_id = null
+
+/datum/aiTask/succeedable/move/New()
+	. = ..()
+	if(isliving(src.holder.owner)) // todo: find out why the hell critters have "implants", which is where their implants are kept
+		var/mob/living/L = src.holder.owner // instead of in mob/living's implant list... and fix it
```

Comment:
?

Reply:
I'm very confused by why mob/living/critter (which uses C.implants) has a separate handling of implants than mob/living (which uses L.implant) and want to fix it at a later date